paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/USBankAccountFormViewModel.kt

```diff
@@ -236,6 +241,9 @@ internal class USBankAccountFormViewModel @Inject internal constructor(
         state.updateWithMandate(mandateText)
     }
 
+    private val _analyticsEvent = MutableSharedFlow<AnalyticsEvent>(replay = 1)
```

Comment:
Would this not risk sending events twice due to the replay logic?

Reply:
yep, and it's actually not needed

Reply:
https://github.com/stripe/stripe-android/pull/10643/commits/706dad05177968d21a177f5b20587725b4aab67f